crates/red_knot_module_resolver/src/typeshed/versions.rs

```diff
@@ -3,10 +3,20 @@ use std::fmt;
 use std::num::{NonZeroU16, NonZeroUsize};
 use std::ops::{RangeFrom, RangeInclusive};
 use std::str::FromStr;
+use std::sync::Arc;
 
+use ruff_db::{source::source_text, vfs::VfsFile};
 use rustc_hash::FxHashMap;
 
-use crate::module::ModuleName;
+use crate::db::Db;
+use crate::module_name::ModuleName;
+use crate::supported_py_version::SupportedPyVersion;
+
+#[salsa::tracked]
+pub(crate) fn parse_typeshed_versions(db: &dyn Db, versions_file: VfsFile) -> TypeshedVersions {
+    let file_content = source_text(db.upcast(), versions_file);
+    file_content.parse().unwrap()
```

Comment:
Unwrapping is obviously the wrong thing to do here... Should I just return a `Result` from this query? Not sure how to handle this. If the user passes us a custom typeshed directory with an invalid `VERSIONS` file, we can't infer any information about the standard library